From e621d6acb538d717d6767429e3f85190019a887d Mon Sep 17 00:00:00 2001
From: Thomas Claveirole <thomas.claveirole@green-communications.fr>
Date: Mon, 18 Jul 2016 15:06:59 +0200
Subject: [PATCH] configure.ac: Check for libsystemd, not libsystemd-daemon.

Newer versions of systemd do not provide libsystemd-daemon anymore.
Therefore, use libsystemd instead.

Signed-off-by: Thomas Claveirole <thomas.claveirole@green-communications.fr>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index bb3674e..2b02ef4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -28,7 +28,7 @@ AC_ARG_WITH([systemd],
   [], [with_systemd=check])
 have_systemd=no
 if test "x$with_systemd" != "xno"; then
-  PKG_CHECK_MODULES(systemd, [libsystemd-daemon],
+  PKG_CHECK_MODULES(systemd, [libsystemd],
     [AC_DEFINE(HAVE_SYSTEMD, 1, [Define if systemd is available])
     have_systemd=yes],
   have_systemd=no)
